content/api.md

```diff
@@ -0,0 +1,59 @@
+# Tier API
+
+# Authentication
+
+1. Visit the [dashboard](https://app.tier.run) and navigate to the `Tokens` page.
+2. Generate an API key and keep it safe.
+3. Provide the key as the user key in basic auth when making calls to the Tier API (leave the password blank)
+
+# APIs
+
+* Plans
```

Comment:
You can just put a `## Table of Contents` heading with no content, and it'll be autogenerated from headings throughout the doc.

Reply:
Scratch that, I'm doing TOCs automatically now.  Just organize the content under headings, tierweb will do the rest.